tools/kochdocs.nim

```diff
@@ -51,6 +51,12 @@ proc execCleanPath*(cmd: string,
   if execShellCmd(cmd) != 0: quit("FAILURE", errorcode)
   putEnv("PATH", prevPath)
 
+let kochExe* = os.getAppFilename()
+  # note: assumes `kochdocs` is only used by koch.nim
```

Comment:
You can easily move ``kochExe`` and ``kochExec`` to ``koch.nim`` btw, but I'm merging this now.

Reply:
=> fixed as part of https://github.com/nim-lang/Nim/pull/10242